src/main/java/io/wispforest/owo/serialization/impl/SerializationAttribute.java

```diff
@@ -0,0 +1,8 @@
+package io.wispforest.owo.serialization.impl;
+
+public enum SerializationAttribute {
+    SELF_DESCRIBING(),
+    BINARY(),
+    HUMAN_READABLE(),
+    COMPRESSED();
```

Comment:
As in compressed with gzip/whatever? Why would a Codeck want to care about this?

Reply:
As in the the data should be as small as possible but still hold onto some form of structure as in not intended to be human readable but still can be read in a sense based on order or something